src/usr/local/www/services_dyndns_edit.php

```diff
@@ -364,6 +364,7 @@ function build_if_list() {
 ))->setHelp('Username is required for all types except Namecheap, FreeDNS and Custom Entries.%1$s' .
 	                'Azure: Enter your Azure AD application ID%1$s' .
 			'DNS Made Easy: Dynamic DNS ID%1$s' .
+			'DNSimple: User account ID (In the URL after the '/a/')%1$s' .
```

Comment:
Does that actually work like you show? It has unescaped single quotes inside single quotes looks like it would throw an error and/or not format that string properly.

Reply:
That syntax error leads me to think that the code was not tested sufficiently. Please perform a regression test of your changes and post hte results here.

Reply:
> That syntax error leads me to think that the code was not tested sufficiently. Please perform a regression test of your changes and post hte results here.

Any pointers on where I can find instructions running regression tests? I have since corrected the issue with escaped quotes in bad3542. The other half of this code is currently running on my local install of pfSense.